gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -197,7 +197,18 @@ MM_VerboseHandlerOutput::getTagTemplate(char *buf, uintptr_t bufsize, uintptr_t
 	if (NULL != reasonForTermination){
 		bufPos += omrstr_printf(buf + bufPos, bufsize - bufPos, " terminationReason=\"%s\"", reasonForTermination);
 	}
+	return bufPos;
+}
+
+uintptr_t
+MM_VerboseHandlerOutput::getTagTemplate(char *buf, uintptr_t bufsize, uintptr_t id, const char *type, uintptr_t contextId, uint64_t timeus, uint64_t wallTimeMs, const char *reasonForTermination)
```

Comment:
I think we can reduce the number of `getTagTemplate` methods being introduced when we have this  `reasonForTermination` pattern. I see it here and I think it was initially introduced in a previous changes where I made the same suggestion...

Instead of creating a new method you can modify the "base" `getTagTemplate` method (being called in line 207 here) to take in `reasonForTermination` (default to NULL) and do the `if` `NULL` check / adding to buffer  there.